airflow/contrib/hooks/aws_athena_hook.py

```diff
@@ -127,14 +127,21 @@ def poll_query_status(self, query_execution_id, max_tries=None):
             elif query_state in self.INTERMEDIATE_STATES:
                 self.log.info('Trial {try_number}: Query is still in an intermediate state - {state}'
                               .format(try_number=try_number, state=query_state))
+            elif query_state in self.FAILURE_STATES:
+                self.log.error('Trial {try_number}: Query is failed - {state}'
+                               .format(try_number=try_number, state=query_state))
+                raise Exception('Athena job {state}.'.formate(state=query_state))
```

Comment:
If you raise the Exception here, no retry will be "retried" ;-) The logic here will be broken.

Reply:
I thought if the query is failed, we don't need to retry to query the state. Retry only happens when `query_state` is 'QUEUED' or 'RUNNING'. Does it make sense?

Reply:
Ha, get what you meant. Let me have another look